ds-prep-python-conditionals/main.py

```diff
@@ -0,0 +1,43 @@
+import random
+
+day_of_week = 'monday'
+print ("It's a weekday!") if day_of_week == 'saturday' or 'sunday' else print ("Have a good weekend!")
```

Comment:
You'll find that this doesn't produce the right output when `day_of_week` is `'sunday'`:

---

![image](https://user-images.githubusercontent.com/16143771/149250127-9d4b76e0-cff7-49d7-8d3f-72023bb0448d.png)